src/main/java/com/snowflake/snowpark_java/DataFrame.java

```diff
@@ -805,14 +805,10 @@ public Column col(String colName) {
   /**
    * Returns the current DataFrame aliased as the input alias name.
    *
-   * For example:
+   * <p>For example:
    *
-   * {{{
-   *   val df2 = df.alias("A")
-   *   df2.select(df2.col("A.num"))
-   * }}}
+   * <p>{{{ val df2 = df.alias("A") df2.select(df2.col("A.num")) }}}
```

Comment:
Why make those two in the same line?

Reply:
Also can we add a example for alias showing use cases like this since we are modifying this part?
```
val df2 = df.alias("A")
df2.select(col("A.num"))
df2.select($"A.num")
```